spec/system/placements/organisations/users/user_removes_other_users_from_organisation_spec.rb

```diff
@@ -0,0 +1,150 @@
+require "rails_helper"
+
+RSpec.describe "Placements support user removes a user from an organisation", type: :system, service: :placements do
+  describe "schools" do
+    let(:school) { create(:placements_school) }
+    let(:anne) { create(:placements_user, :anne) }
+    let(:mary) { create(:placements_user, :mary) }
+
+    before do
+      [anne, mary].each do |user|
+        create(:membership, user:, organisation: school)
+      end
+    end
+
+    scenario "user is removed from a school" do
+      given_i_am_signed_in_as_mary
+      and_i_visit_the_user_page(anne)
+      when_i_click_on("Remove user")
+      then_i_am_asked_to_confirm(anne, school)
+      when_i_click_on("Cancel")
+      then_i_return_to_user_page(anne, school)
+      when_i_click_on("Remove user")
+      then_i_am_asked_to_confirm(anne, school)
+      when_i_click_on("Remove user")
+      then_the_the_user_is_removed_from_the_organisation(anne, school)
+      and_message_is_sent_to_user(anne, school)
+    end
+
+    scenario "I try to remove myself from a school" do
+      given_i_am_signed_in_as_mary
+      when_i_try_to_visit_the_remove_path_form_myself(mary, school)
+      then_i_am_redirected_to_the_index_page(school)
```

Comment:
Would be sensible to additionally assert that the "Remove user" link doesn't show?

```suggestion
      given_i_am_signed_in_as_mary
      and_i_visit_the_user_page(mary)
      then_i_cannot_see("Remove user")
      when_i_try_to_visit_the_remove_path_form_myself(mary, school)
      then_i_am_redirected_to_the_index_page(school)
```

**Note:** `then_i_cannot_see` is made up!

Reply:
tested in the other spec! :)